docs/pages/prerequisites.md

```diff
@@ -0,0 +1,27 @@
+# Prerequisites
+
+To get started with Fable, you will need a couple of things installed on your machine. 
+
+ - F# Compiler
+ - Node (with Npm)
+ - Yarn
+ - Dotnet SDK
+ - IDE
+
+## F# Compiler
+If you have Visual Studio or Xamarin installed with Visual F# Tools installed then you already have F# Compiler on your machine. If you are starting from scratch, refer to [fsharp.org](http://fsharp.org/) to get started with F#. 
+
+## Node
+Node is a javascript runtime. That means Node to javascript is what .NET is to F#. Node is central to the javascript ecosystem. You can insall Node from the [official website](https://nodejs.org/en/). You need it becuase Fable has dependencies that run on Node. Other than that, you could compile your F# code and run it yourself using Node.  
+
+## Npm (Node Package Manager)
```

Comment:
Should we even mention this explicitly since it comes with the node install and we'll be recommending yarn?